src/dir.targets

```diff
@@ -11,5 +11,6 @@
           Returns="$(AssemblyVersion)"/>
 
   <Import Project="$(ToolsDir)Build.Common.targets" Condition="Exists('$(ToolsDir)Build.Common.targets')" />
+  <Import Project="..\override.targets" Condition="Exists('..\override.targets')"/>
```

Comment:
Do you intend for the override.targets file to stay around and become part of the new convention or repo's using buildtools? If not I would just import the mono.targets in this file directly.